src/components/marker-table/ContextMenu.js

```diff
@@ -113,16 +121,31 @@ class MarkersContextMenu extends PureComponent<Props> {
 
   copyMarkerJSON = () => {
     const { selectedMarker, markers } = this.props;
+
+    if (selectedMarker === null) {
+      return;
+    }
+
```

Comment:
As I reminder, we had a discussion about these null checks in https://github.com/firefox-devtools/profiler/pull/1683#discussion_r251482015

In short, Flow makes it harder than necessary to create a "ContextMenuContents" component where the marker wouldn't be null.